squad/core/models.py

```diff
@@ -1126,13 +1121,6 @@ def __init__(self, build, environment=None):
                 self.tests_xfail += 1
             elif test.status == 'skip':
                 self.tests_skip += 1
-            else:
-                raise InvalidTestStatus(test.status)
-            if test.status == 'fail':
```

Comment:
I think this is used in email templates. At least it can be.

Reply:
ah, that's why is good to have another pair of eyes, replaced it!